src/settings/main.cpp

```diff
@@ -395,7 +398,7 @@ void quit_when_parent_terminates(std::wstring parent_pid) {
   std::thread(wait_on_parent_process_thread, pid).detach();
 }
 
-void initialize_message_pipe() {
+void initialize_args() {
```

Comment:
`initialize_args` is not a good name for this function. Before this change the only thing it was doing was to initialize the message pipe, and it got the name from there, now that it also sets the dark mode we have to call it something else like "parse_args` and put a comment to describe the it parses the arguments to initialize the message pipe and set the dark mode.